readme.md

```diff
@@ -117,3 +117,17 @@ user.save()
 ```
 
 8. You can now access the /admin page.
+
+## Setup pre-commit for identify simple issues and standardize code formatting
+
+Pyladiescon-portal uses a tool called [pre-commit](https://pre-commit.com/) to identify simple issues and standardize code formatting. It does this by installing a git hook that automatically runs a series of code linters prior to finalizing any git commit. To enable pre-commit, run:
+
+```
+pre-commit install
+```
+
+### Pre-commit automatically runs during the commit
+
+With pre-commit installed as a git hook for verifying commits, the pre-commit hooks configured in `.pre-commit-config.yaml` for Pyladiescon-portal must all pass before the commit is successful. If there are any issues found with the commit, this will cause your commit to fail. Where possible, pre-commit will make the changes needed to correct the problems it has found. 
```

Comment:
```suggestion
With pre-commit installed as a git hook for verifying commits, the pre-commit hooks configured in `.pre-commit-config.yaml` for PyLadiescon-portal must all pass before the commit is successful. If there are any issues found with the commit, this will cause your commit to fail. Where possible, pre-commit will make the changes needed to correct the problems it has found. 
```